.travis.yml

```diff
@@ -21,7 +21,7 @@ addons:
     - mongodb-3.0-precise
     packages:
     - mongodb-org-server
-  postgresql: "9.5"
+  postgresql: "9.6"
```

Comment:
Could still support Postgres 9.5 if we add a `POSTGRES_RAISE_ON_ERROR` setting that could be set to `False` and it would be assumed that the database is managed manually.